doc/changelog/04-tactics/09288-injection-as.rst

```diff
@@ -0,0 +1,4 @@
+- Documented syntax :n:`injection @term as [= {+ @intropattern} ]` as
+  an alternative to :n:`injection @term as {+ @simple_intropattern}` using
+  the standard :token:`@injection_intropattern` syntax (`#09288
```

Comment:
It can be a bit confusing, but `:token:` only takes the name of the token, like this:
```suggestion
  the standard :token:`injection_intropattern` syntax (`#09288
```

but if you want to always use the `@` syntax, this is the alternative (both render the same):

```suggestion
  the standard :n:`@injection_intropattern` syntax (`#09288
```

Reply:
Done.